src/parser/tbc/modules/items/WeaponEnhancementChecker.tsx

```diff
@@ -8,6 +8,7 @@ const MAX_ENHANCEMENT_IDS = [
   2713, // https://tbc.wowhead.com/spell=29453/sharpen-blade
   2955, // https://tbc.wowhead.com/spell=34340/weight-weapon
   2629, // https://tbc.wowhead.com/spell=25123/brilliant-mana-oil
+  2639, // https://tbc.wowhead.com/spell=25587/windfury-totem
```

Comment:
This will work if the fight starts out and you have a windfury totem down, but we will need to create an analyzer that checks to see if you ever have it up to know if we should show the missing enhancement suggestion.